syntax = "proto3";

option java_package = "self.grpc";

package helloworld;

service Greeter {
    rpc SayHello(HelloRequest) returns (HelloReply) {}
    rpc GetObject(Empty) returns (HelloObj) {}
}

message HelloRequest {
    string name = 1;
}

message HelloReply {
    string message = 1;
}

message HelloObj {
    string name = 1;
    string email = 2;
    int32 id = 3;
}

message Empty {}
